clientv3/integration/kv_test.go

```diff
@@ -110,6 +110,29 @@ func TestKVPut(t *testing.T) {
 	}
 }
 
+func TestKVPutWithRequireLeader(t *testing.T) {
+	defer testutil.AfterTest(t)
```

Comment:
add t.Parallel() to make this in parallel with other tests. or it will wait for a few seconds and block other tests.